blocks/cards/cards.css

```diff
@@ -1,19 +1,43 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap');
+
+.cards-container {
+    padding: 0 !important;
+    margin: 0;
+    box-sizing: border-box;
+    color: white;
+}
+
 .cards > ul {
     list-style: none;
-    margin: 0;
+    margin: auto;
     padding: 0;
-    display: grid;
-    grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
-    grid-gap: 16px;
+    display: flex;
+    flex-wrap: nowrap;
+    width: 100%;
+    background-image: url(https://www.partsasist.com/-/media/themes/aftermarket/partsasist/main-website/default/images/featurecard_bg.jpg);
```

Comment:
This image should be imported and referenced via the (this) Franklin site

Reply:
Ok. This I wasn't sure how to do. I'll check tomorrow how to do it correctly